README.md

```diff
@@ -85,3 +85,24 @@ First [retrieve your Discord User Token](https://discordhelp.net/discord-token).
 
 Then send the bot a message with the contents `link your.token-here`.
 
+## Docker
+
+Build docker image:
+
+    docker build -t mx-puppet-discord .
+
+You may want some changes in your config.yaml:
+
+```yaml
+bindAddress: 0.0.0.0
+blacklist: []
```

Comment:
Maybe change that blacklist is commented out by default in the default config instead?

Reply:
sure